insights/parsers/du_candlepin_activemq_artemis.py

```diff
@@ -0,0 +1,23 @@
+"""
+DuVarLibCandlepinActivemqArtemis - command ``/bin/du -s -k /var/lib/candlepin/activemq-artemis``
+================================================================================================
+
+"""
+from insights import DiskUsage, parser
+from insights.specs import Specs
+
+
+@parser(Specs.du_candlepin_activemq_artemis)
+class DuVarLibCandlepinActivemqArtemis(DiskUsage):
+    """
+    Parser to get disk usage of "/var/lib/candlepin/activemq-artemis" directory
+
```

Comment:
Please provide sample input data here to make it clear where the data in the examples comes from.

Reply:
removed the child parser class and implemented the combiner approach